js/packages/phoenix-client/src/types/datasets.ts

```diff
@@ -0,0 +1,21 @@
+import { Node } from "./core";
+
+/**
+ * An example is a record to feed into an AI task
+ */
+export interface Example extends Node {
+  id: string;
```

Comment:
not needed cause it extends node, same with dataset below

Reply:
👍🏻